internal/compiler-bridge/src-2.10/main/scala/xsbt/CompilerInterface.scala

```diff
@@ -218,8 +218,10 @@ private final class CachedCompiler0(args: Array[String], output: Output, initial
     override lazy val phaseDescriptors =
       {
         phasesSet += sbtAnalyzer
```

Comment:
You say about `AnalysisCallback#enabled` "If returns false, no sbt phases will be introduced in compiler". Why is `sbtAnalyzer` still added?